### PR TITLE
Update automated-patching.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

**Comment by ujpat** [comment]
@bluefooted for review

**Comment by American-Dipper** [comment]
@tarynpratt - can you review the proposed changes?  

 
IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.  

 

#label:"aq-pr-triaged" 

@MicrosoftDocs/public-repo-pr-review-team

**Review by tarynpratt — APPROVED** [review]

**Comment by tarynpratt** [comment]
#sign-off

File: azure-sql/virtual-machines/windows/automated-patching.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ To disable Automated Patching, run the following script with the value of **$fal
 Update-AzSqlVM -ResourceGroupName 'resourcegroupname' -Name 'vmname' -AutoPatchingSettingEnable:$false
 ```
 
+## Understanding What Updates will be applied with Automated Patching
+Please ensure to review the [Update Guide](https://msrc.microsoft.com/update-guide) and apply the **Severity** filter to identify Critical and Important updates in order to understand the updates that will be applied through Automated Patching.
```
Comment (ShannonLeavitt) [inline azure-sql/virtual-machines/windows/automated-patching.md]:
```suggestion
## Understand which updates will be applied with Automated Patching
To understand which updates will be applied through Automated Patching, review the [update guide](https://msrc.microsoft.com/update-guide) and apply the **Severity** filter to identify Critical and Important updates.
```
Minor edits for Microsoft style

**Pull request merged**